src/SonarScanner.MSBuild.Common/SonarProperties.cs

```diff
@@ -57,5 +57,8 @@ public static class SonarProperties
 
         public const string VsCoverageXmlReportsPaths = "sonar.cs.vscoveragexml.reportsPaths";
         public const string VsTestReportsPaths = "sonar.cs.vstest.reportsPaths";
+
+        public const string Proxy = "sonar.proxy.host";
+        public const string ProxyCredencial = "sonar.proxy.credencial";
```

Comment:
I think both the name and the value of the constant should be `credential` (with `t`)

Reply:
Done